src/test/spec/crud.rs

```diff
@@ -18,6 +18,12 @@ fn test_predicate(test: &TestCase) -> bool {
     // The Rust driver doesn't support unacknowledged writes.
     let lower = test.description.to_lowercase();
 
-    // TODO: RUST-1071, RUST-1215: unskip comment tests
-    !lower.contains("unacknowledged") && !lower.contains("comment")
+    // TODO: RUST-1071, RUST-663: unskip comment tests
+    // and aggregate $out and $merge tests
+    !lower.contains("unacknowledged")
+        && (!lower.contains("comment")
+            || (lower.contains("estimated")
+                && lower.contains("document")
+                && lower.contains("count")))
+        && !(lower.contains("aggregate") && lower.contains("preference"))
```

Comment:
It's possible that there are unrelated CRUD tests that contain both "aggregate" and "preference" in their descriptions, so I think we should list out the exact descriptions for the tests we want to skip here to ensure we're not losing any existing coverage.